commerce-account-service/src/main/java/com/liferay/commerce/account/internal/util/CommerceAccountHelperImpl.java

```diff
@@ -127,6 +130,26 @@ else if (currentCommerceAccountId > 0) {
 		return commerceAccount;
 	}
 
+	@Override
+	public long[] getUserCommerceAccountIds(long userId, long groupId)
+		throws PortalException {
+
+		List<CommerceAccount> commerceAccounts =
+			_commerceAccountLocalService.getUserCommerceAccounts(
+				userId, CommerceAccountConstants.DEFAULT_PARENT_ACCOUNT_ID,
+				_getCommerceSiteType(groupId), StringPool.BLANK,
+				QueryUtil.ALL_POS, QueryUtil.ALL_POS);
+
+		long[] commerceAccountIds = new long[0];
```

Comment:
Sort `long[] commerceAccountIds =...` above `List<CommerceAccount> commerceAccounts =...`

Reply:
Actually, just return the `ListUtil.toLongArray` call -- the implementation already checks for null/empty lists, so we don't need to double check before calling it. Then we don't need the `long[]` variable at all.